test/functional/api/cas/cache.py

```diff
@@ -66,6 +65,22 @@ def get_status(self):
         status = self.get_statistics().config_stats.status.replace(' ', '_').lower()
         return CacheStatus[status]
 
+    @staticmethod
+    def get_metadata_max_size(cache_size: Size, cache_line_size: Size, number_of_cores=4096):
+        """Get memory size according to requirement """
+        constant = int(Size(100, Unit.MebiByte))
+        cache_size_value = int(cache_size.get_value(Unit.MebiByte))
+        cache_line_size_value = int(cache_line_size.get_value(Unit.MebiByte))
+
+        cache_line_count = cache_size_value / cache_line_size_value
+        per_cache_line_metadata = 68 + 2 * cache_line_size_value / number_of_cores
+        amount_of_dram = constant + cache_line_count * per_cache_line_metadata
+
+        # maximum value is 110% of calculated size
+        metadata_max_size = 1.10 * amount_of_dram
```

Comment:
It would be great to assign all those hard coded numbers to some constant variables with descriptive names or at least add some comments for clarification, where those numbers came from.

Reply:
Sadly they came from OCF source code, when we use exactly the same formula. We have no better names for those constants, this is partially empirical data.

Reply:
Let's attach your response as a comment here then. ;)